bim2sim/examples/e1_interactive_project.py

```diff
@@ -2,16 +2,17 @@
 from pathlib import Path
 
 from bim2sim import Project, ConsoleDecisionHandler
-from bim2sim.log import default_logging_setup
+from bim2sim.kernel.log import default_logging_setup
 from bim2sim.utilities.types import IFCDomain
 
 
-def run_example_2():
+# TODO #548 Implement two examples which don't use any "third party" plugins
+def run_example_1():
     """Run the building simulation with teaser as backend in interactive mode.
     
     Interactive mode means that we use open_conf=True to open up the config.toml
     during the process to change settings and use an interactive PlayGround 
-    which allows us to select which task we want to proceed with after a task
+    which allows us to select which tasks we want to proceed with after a tasks
     is finished and don't use the predefined order of default_tasks for the 
     selected Plugin. 
     """
```

Comment:
Please add general comments on the following three lines (they still refer to e1, which does no longer exist. Please also update the prefix in the project path (Line 22).

Reply:
adjusted the example and made clear that this is WIP as we need the base plugins before this. Already started with them and opened added issues to solve here #548